### PR TITLE
Use standard ctapipe camera names

### LINKED ISSUE #59: Use standard ctapipe camera names
See cta-observatory/image-extractor/issues/15.

### PR DESCRIPTION
**Pull request by bryankim96**
Closes #59.

Updated all telescope type names to match ctapipe telescope type names. Added temporary logic for backwards compatibility with old data files (pre-v0.6.0 versions of ImageExtractor).

Separated camera-dependent parameters (image size, bounding box, padding, etc.) from telescope-dependent parameters. Added get_camera_type method to extract camera type name from ctapipe telescope name string.

Updated default models to match new usage of camera type (added telescope_type_to_camera_type metadata field for easy lookup).

### CODE REVIEW

File: ctlearn/data_loading.py [hunk 1]
```diff
@@ -9,18 +9,32 @@
 import tables
 
 from ctlearn.data_processing import DataProcessor
-from ctlearn.image_mapping import ImageMapper
+from ctlearn.image_mapping import ImageMapper, get_camera_type
+
+# TEMPORARY: Maps old telescope names into new
+# For training with datasets generated with versions prior to image-extractor v0.6.0
+OLD_TEL_NAMES_TO_NEW = {
```
Comment (aribrill) [inline ctlearn/data_loading.py]: These dicts are defined twice, in data_loading and image_mapping. Maybe define them just in image_mapping and import them.

File: config/example_config.yml [hunk 2]
```diff
@@ -40,27 +40,27 @@ Data:
...
+        #   - 'SST:ASTRICam': SST-2M ASTRI Telescope and Camera
+        #   - 'SST:CHEC': SST-2M GCT Telescope with CHEC Camera
+        #   - 'MST:VERITAS': VERITAS Telescope and Camera
         # List of telescope types to load.
-        selected_tel_types: ['LST']
-
-        # Optional dictionary with string keys and list of integer values,
+        selected_tel_types: ['LST:LSTCam']
+        
+            # Optional dictionary with string keys and list of integer values,
```
Comment (aribrill) [inline config/example_config.yml]: Looks like an extra indent here. Consider running the config file through yamllint if you haven't already to catch any style issues.

Comment (bryankim96) [reply config/example_config.yml]: Checked with a linter and fixed in a278272d4ed21f6a3f67fbcfbc9bf3e021ca1bca.

File: ctlearn/data_loading.py (hunk 1 above)
Comment (bryankim96) [reply ctlearn/data_loading.py]: Removed in c69382f22114aacdaba257c89f0b8d96ea334b0e.

File: config/example_config.yml [hunk 3]
```diff
@@ -35,8 +35,14 @@ Data:
         example_type: 'array'
 
         # Optional Boolean. Default: False
+<<<<<<< HEAD
```
Comment (aribrill) [inline config/example_config.yml]: Merge message accidentally left in.

Comment (bryankim96) [reply config/example_config.yml]: My mistake, not quite sure how that happened (my editor checks for conflicts before doing a merge commit).

**Pull request merged**